stdlib/shutil.pyi

```diff
@@ -117,7 +117,11 @@ else:
     def unpack_archive(filename: str, extract_dir: StrPath | None = ..., format: str | None = ...) -> None: ...
 
 def register_unpack_format(
-    name: str, extensions: list[str], function: Any, extra_args: Sequence[tuple[str, Any]] | None = ..., description: str = ...
+    name: str,
+    extensions: list[str],
+    function: Callable[..., object],
```

Comment:
It should be approximately `Callable[[str, str], object]`, where the first arg is the filename and the second is the target directory. But it also passes whatever is in `extra_args` as **kwargs to the unpack function. I don't think we can express that accurately, so let's stick with `...`.

Reply:
I thought about using an `@overload` for case when `extra_args` are `None`, but this might be an over-kill.

Reply:
Yes, that works if you're interested in making this change.

Reply:
I guess I have to now 🙂

Reply:
You don't have to, I'm happy to merge this change as is if you prefer.

But better types are, well, better :)